packages/instant/src/util/buy_quote_updater.ts

```diff
@@ -31,6 +37,13 @@ export const buyQuoteUpdater = {
         } catch (error) {
             if (options.dispatchErrors) {
                 dispatch(actions.setQuoteRequestStateFailure());
+                analytics.trackQuoteError(
+                    error.message ? error.message : 'other',
+                    asset.metaData.name,
+                    asset.assetData,
+                    assetUnitAmount,
```

Comment:
Let's log the asset amount in wei (base units) because we're logging wei amounts for buyQuoteProperties

Reply:
Good call that these should be consistent. I'm wondering though: would it be better to instead convert the `buyQuoteProperties` to report in unit amounts instead?  I feel like we usually think in these amounts, and it'd be easier to not have to do the math when viewing items.  Let me know what you think.

Reply:
I think its better to report base units because we will need additional context of token decimals to get the true amount being bought.